java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/MoveClassUI.java

```diff
@@ -305,7 +305,11 @@ public RefactoringUI create(CompilationInfo info, TreePathHandle[] handles, File
                 Collection<TreePathHandle> tphs = new ArrayList<TreePathHandle>();
                 SourcePositions sourcePositions = info.getTrees().getSourcePositions();
                 for (Element ele : e.getEnclosedElements()) {
-                    Tree leaf = info.getTrees().getPath(ele).getLeaf();
+                    TreePath path = info.getTrees().getPath(ele);
+                    if (path == null) {
```

Comment:
I assume this is null for the synthetic record members, etc., right?

Reply:
Yes, Its for synthetic record members.